### DISCUSSION TITLE
Writing/reading encoding

### DISCUSSION THREAD

**Discussion by Tyoma7Tmp** [nlohmann/json#3046] [Q&A] [answered]
Good afternoon, I would like to know about the encoding.
I kwon about "only UTF-8", but I need to write data in Cyrillic (RUS symbols) and subsequently read them. I tried it already and it does not work, an exception is thrown. How can I do this?

**Comment by nlohmann** [comment, marked as answer]
See https://github.com/nlohmann/json#character-encoding and https://json.nlohmann.me/home/faq/#parse-errors-reading-non-ascii-characters

**Reply by Tyoma7Tmp** [reply]
Thank you. We will look and understand.
